index.src.html

```diff
@@ -858,66 +601,22 @@ <h3 id="requirements-user-controls">User Controls</h3>
     [[RFC6919]] present such a choice to users without careful consideration and
     communication of the risk involved.
 
+    A user agent MAY offer users the ability to override its decision to
+    automatically upgrade <a>upgradeable</a> mixed content on a
+    particular page.
```

Comment:
I'd rather not have this, but I guess I should open an issue for that.

Reply:
I'd appreciate if you do. I lean towards having the opt-out ability documented, but I might be biased since Chrome's implementation includes an opt-out. I can be convinced that this is not needed in the spec, but the conversation would be more visible for posterity in an issue.

Reply:
I guess if Chrome still includes an opt-out I'm not going to bother.